nafas/functions.py

```diff
@@ -4,7 +4,8 @@
 import time
 from nafas.params import NAFAS_DESCRIPTION, NAFAS_NOTICE, STANDARD_MENU, STANDARD_MENU_ORDER, STEP_MAP
 from nafas.params import PROGRAMS, PROGRAM_DESCRIPTION, SOUND_MAP, STEP_TEMPLATE, CYCLE_TEMPLATE
-from nafas.params import SOUND_WARNING_MESSAGE, EXIT_MESSAGE
+from nafas.params import SOUND_WARNING_MESSAGE, EXIT_MESSAGE, BAD_INPUT_MESSAGE, PROGRAM_END_MESSAGE
+from nafas.params import MINUTES_TEMPLATE, SECONDS_TEMPLATE, PROGRAM_TIME_TEMPLATE, MENU_TEMPLATE_1, MENU_TEMPLATE_2
```

Comment:
Move the import of `MENU_TEMPLATE_1` and `MENU_TEMPLATE_2` to the next line. It's too long now.

Reply:
fixed in https://github.com/sepandhaghighi/nafas/pull/105/commits/8e4bc8465a32b3dfe822cf509f7d2511754b974e